deploy/helm/nfd-operator/crds/nfd-api-crds.yaml

```diff
@@ -0,0 +1,686 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: unapproved, experimental-only
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: nodefeaturediscoveries.nfd.kubernetes.io
+spec:
+  group: nfd.kubernetes.io
+  names:
+    kind: NodeFeatureDiscovery
+    listKind: NodeFeatureDiscoveryList
+    plural: nodefeaturediscoveries
+    singular: nodefeaturediscovery
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        description: NodeFeatureDiscovery is the Schema for the nodefeaturediscoveries
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: NodeFeatureDiscoverySpec defines the desired state of NodeFeatureDiscovery
+            properties:
+              extraLabelNs:
+                description: ExtraLabelNs defines the list of of allowed extra label
+                  namespaces By default, only allow labels in the default `feature.node.kubernetes.io`
+                  label namespace
+                items:
+                  type: string
+                nullable: true
+                type: array
+              instance:
+                description: Instance name. Used to separate annotation namespaces
+                  for multiple parallel deployments.
+                type: string
+              labelWhiteList:
+                description: LabelWhiteList defines a regular expression for filtering
+                  feature labels based on their name. Each label must match against
+                  the given reqular expression in order to be published.
+                nullable: true
+                type: string
+              operand:
+                description: OperandSpec describes configuration options for the operand
+                properties:
+                  image:
+                    description: Image defines the image to pull for the NFD operand
+                      [defaults to k8s.gcr.io/nfd/node-feature-discovery]
```

Comment:
Outdated. You probably want to sync from `config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml`